attestation-service/verifier/src/tdx/quote.rs

```diff
@@ -528,16 +528,40 @@ mod tests {
         let _ = fs::write(format!("{quote_path}.txt"), parsed_quote);
     }
 
+    // TODO: replace the file read-write operation to ENV settings once
+    // https://github.com/intel/SGXDataCenterAttestationPrimitives/issues/409
+    // gets fixed.
+    const SGX_QCNL_CONFIG_PATH: &str = "/etc/sgx_default_qcnl.conf";
+    const SGX_QCNL_CONFIG: &str =
+        r#"{"pccs_url" :"https://sgx-dcap-server.cn-beijing.aliyuncs.com/sgx/certification/v4/"}"#;
+
+    // This test requires two packages, s.t. libsgx-dcap-quote-verify-dev libsgx-dcap-default-qpl
+    // See attestation-service/Dockerfile.as-grpc to check how to install the two packages
+    //
+    // Also, DCAP only provides packages on x86-64 platform, thus we only test this on x86-64
+    // platforms.
+    #[cfg(target_arch = "x86_64")]
     #[rstest]
-    #[ignore]
     #[tokio::test]
     #[case("./test_data/tdx_quote_4.dat")]
-    #[ignore]
     #[tokio::test]
     #[case("./test_data/tdx_quote_5.dat")]
     async fn test_verify_tdx_quote(#[case] quote: &str) {
+        use std::path::Path;
+
         let quote_bin = fs::read(quote).unwrap();
+        let mut original_content = None;
+
+        if Path::new(SGX_QCNL_CONFIG_PATH).exists() {
+            original_content = Some(fs::read(SGX_QCNL_CONFIG_PATH).unwrap());
+        }
+        let _ = fs::write(SGX_QCNL_CONFIG_PATH, SGX_QCNL_CONFIG).unwrap();
```

Comment:
I'm not sure it's a good idea to write a global config file to a machine which is running the unix test, I wouldn't like that a as a developer who is checking out the repo, even if we attempt to write an existing file back. I would rather add instructions to the documentation block and skip this test, until we have dynamic config file paths

Reply:
Agreed. Let's make it still ignored.